### PR TITLE
Proposal for the Polish translation of the 'Download Kubernetes' section

### PR DESCRIPTION
**Pull request by adampuchejda**
Proposal for the Polish translation of the 'Download Kubernetes' section.

PL translation of the 'Download Kubernetes' (download.md) section. Pushing to the main branch in the absence of the dev-pl localization branch.

### CODE REVIEW

File: content/pl/releases/download.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Dla wersji {{< skew currentVersion >}} Kubernetesa jedynym typem artefaktu kodu, którego integralność możesz zweryfikować, jest obraz kontenera (korzystając z eksperymentalnej opcji podpisu). 
+
+By ręcznie zweryfikować podpisane obrazy kontenerów głównych komponentów Kubernetesa, zobacz [Zweryfikuj podpisane obrazy kontenerów](/docs/tasks/administer-cluster/verify-signed-artifacts).
+
+## Pliki binarne
+
+Odnośniki do plików binarnych z komponentami Kubernetesa (i ich sumy kontrolne) znajdziesz w [CHANGELOGU](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG).
+
+Możesz też skorzystać ze strony [downloadkubernetes.com](https://www.downloadkubernetes.com/), gdzie szukane komponenty możesz filtrować w zależności od wersji i architektury sprzętowej.
```
Comment (sftim) [inline content/pl/releases/download.md:40]:
```suggestion
{{< release-binaries >}}
```
(and then verify that you've added all the relevant strings into [`data/i18n/pl/pl.toml`](https://github.com/kubernetes/website/blob/main/data/i18n/pl/pl.toml))

**Review by sftim — COMMENTED** [review]
Thanks, but: this isn't up to date with the English original.

File: content/pl/releases/download.md (hunk 1 above)
Comment (adampuchejda) [reply content/pl/releases/download.md]: Hello @sftim! Thank you for catching this difference; I had clearly overlooked the change in the English version. I've made the correction and also included an updated and cleaned Polish version of the i18n file (/data/i18n/pl/pl.toml). @mfilocha, please take a look as well! Very best, Adam

**Comment by mfilocha** [comment]
Thanks, @adampuchejda! Could you please squash all commits into one: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

File: content/pl/releases/download.md (hunk 1 above)
Comment (sftim) [reply content/pl/releases/download.md]: Are you happy with https://deploy-preview-43952--kubernetes-io-main-staging.netlify.app/pl/releases/download/#pliki-binarne? I think there might be some strings missing.

Comment (adampuchejda) [reply content/pl/releases/download.md]: This should change with the new data/i118n/pl/pl.toml file pushed which will happen shortly. I just need to do some cleaning.

**Comment by adampuchejda** [comment]
@mfilocha Now, it should be fine. Thanks :)

File: content/pl/releases/download.md (hunk 1 above)
Comment (sftim) [reply content/pl/releases/download.md]: (still looks like missing strings to me - look for English text)

Comment (adampuchejda) [reply content/pl/releases/download.md]: Thanks for patience. With this addition github.com/kubernetes/website/pull/44054 the release-binaries part should be fine. Please check in your spare time.

**Pull request merged**